### DISCUSSION TITLE
Is there a plan to support ESP32S3 I2S driver

### DISCUSSION THREAD

**Discussion by sam131208** [zephyrproject-rtos/zephyr#68722] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
In product development, sometimes it is necessary to have some chime, or a piece of music, to help enhance the user-friendliness of the product.ESP32 has proven excellent capabilities in this area in IDF, but has not seen any progress in Zephyr. In fact, zephyr already supports LC3, but lacks I2S driver support.

**Describe the solution you'd like**
I2S driver support is available.

**Comment by rftafas** [comment, marked as answer]
Hi,

There was a WIP, but it stalled: https://github.com/zephyrproject-rtos/zephyr/pull/62155

If you need it and are willing to get hands on, Lucas added some suggestion where the previous developer was stuck. 

As for roadmap: we are focused on hal5.1, so I2S is out of question until it gets released. Our expectation is that we will put out a PR in the upcoming days with expectation for it to be approved after the feature freeze window, pushing it to late February or early March, if all goes well. Thus, it pushes I2S (or any new feature) for Q2 or later, and only if it makes the cut against other features being requested (like C6, Low Power, etc).
